test/runtests.jl

```diff
@@ -68,6 +68,173 @@ end
                    8.848886704358202, 2.8697881579099085, 11.15107375630744, 11.8392578374927])
 end
 
+@testset "rankdeficient-glm" begin
+    df = CSV.read("rankdeficient-glm.csv")
+
+    OLS_1 = fit(
```

Comment:
I think the style in this package is to have the first argument on the same line as the function, and other arguments aligned with it.